docs/pages/programming/sql.md

```diff
@@ -338,23 +365,45 @@ will no longer be callable from SQL.
-change not incremental like in other traditional databases.  There's no need to alter the table multiple times to
-add several indices.  The ```ALTER TABLE``` statement will try to form the most efficient to change the old table
-definition to the new one.  Any fields present in the new schema but absent from the old will be added to the 
-table definition.  Any fields absent in the new schema but present int the old will be dropped.  Keys will be added,
-renamed, or changed.  Constraints will be added or removed. See the [Schema definition](table_schema.html) section
-for details on the table schema definition syntax.  See the [Table options](#table-options) section a list of options 
-that may be set for a table.
-
-**All** schema changes in Comdb2 are always live.  The database will not acquire long duration table locks during 
-the change and may be freely read from and written to.  If the schema change adds a new field, or grows the size 
-of an existing field, and doesn't modify the table keys, the change is "*instant*" (unless the ```ISC``` table 
-option is set to ```OFF```).  No table rebuild will take place (unless the table option ```REBUILD``` is specified) 
-if it's not needed.  If fields are removed or the size of an existing field is reduced, the schema change will 
-need to rebuild the existing table.  If any key is modified, it'll be rebuilt.
+ALTER TABLE (II):
+
+![ALTER](images/alter-table-ddl.gif)
+
+The ```ALTER TABLE``` statement will change the definition of the named table
+to the one provided. Note that Comdb2 support two variants of ```ALTER TABLE```
+syntax.
+
+The first approach uses a [declarative language](table_schema.html) and is not
+incremental like SQL. In this approach there is no need to alter the table
+multiple times to add several indices. The ```ALTER TABLE``` statement will try
+to form the most efficient to change the old table definition to the new one.
+Any fields present in the new schema but absent from the old will be added to
+the table definition. Any fields absent in the new schema but present int the
+old will be dropped. Keys will be added, renamed or changed. Constraints will
+be added or removed. See the [Schema definition](table_schema.html) section for
+details on the table schema definition syntax. See the [table options](#table-options)
+section a list of options that may be set for a table.
+
+The second approach, added in version 7.0, supports the usual standard data
+definition language, like other relational database systems. This syntax can
+be used to ```ADD``` a new column or ```DROP``` an existing column from the
+table. Multiple ADD/DROP operations can be used in the same command. In case of
+```DROP``` operation, the references to the column begin dropped will be
+silently removed from the referring keys and constraints definitions.
+
+**Schema changes in Comdb2 are always live**. The database will not acquire
```

Comment:
Make it the 1st paragraph of Alter Table section? Superior to other database systems, schema change in Comdb2 was designed with the goals of being live, least disruptive to normal database operations, and resilient to database crashes. In addition, certain types of schema change can even be completed instantly. It is one of the most exciting features of Comdb2. We might want to mention it first.